engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/details/views/repository-info.html

```diff
@@ -20,6 +20,13 @@ <h4 translate>Basic Information</h4>
       <dt translate>Type</dt>
       <dd>{{ repository.content_type }}</dd>
 
+      <dt translate>Arches</dt>
+      <dd bst-edit-select="repository.arch"
+          selector="repository.arch"
+          options="architectures()"
+          on-save="save(repository)">
+      </dd>
```

Comment:
You probably wnat to put an 'ng-show="repository.content_type === 'yum''  on these elements